controllers/managedocs_controller.go

```diff
@@ -128,13 +130,16 @@ func (r *ManagedOCSReconciler) SetupWithManager(mgr ctrl.Manager) error {
 			},
 		),
 	)
-	addonDeleteConfigMapPredicates := builder.WithPredicates(
+	configMapPredicates := builder.WithPredicates(
```

Comment:
Since there are multiple ConfigMaps, I refactored this predicate to watch for both of them.